package.json

```diff
@@ -22,31 +22,32 @@
 		"firebase": "^9.9.2",
 		"firebase-admin": "^11.0.1",
 		"lodash": "^4.17.21",
-		"next": "12.2.4",
-		"react": "18.2.0",
+		"next": "^12.2.4",
+		"react": "^18.2.0",
 		"react-confetti": "^6.1.0",
-		"react-dom": "18.2.0",
+		"react-dom": "^18.2.0",
 		"react-google-recaptcha": "^2.1.0",
 		"react-player": "^2.10.1",
 		"react-scroll-parallax": "^3.3.1",
 		"sass": "^1.54.3",
+		"swiper": "^8.4.2",
 		"yup": "^0.32.11",
 		"yup-phone": "^1.3.2"
 	},
 	"devDependencies": {
 		"@commitlint/cli": "^17.0.3",
 		"@commitlint/config-conventional": "^17.0.3",
 		"@svgr/webpack": "^6.3.1",
-		"@types/node": "18.6.4",
-		"@types/react": "18.0.16",
-		"@types/react-dom": "18.0.6",
+		"@types/node": "^18.6.4",
+		"@types/react": "^18.0.16",
+		"@types/react-dom": "^18.0.6",
 		"@types/react-google-recaptcha": "^2.1.5",
-		"eslint": "8.21.0",
-		"eslint-config-next": "12.2.4",
+		"eslint": "^8.21.0",
+		"eslint-config-next": "^12.2.4",
 		"husky": "^8.0.1",
-		"lint-staged": ">=10",
-		"prettier": "2.7.1",
-		"typescript": "4.7.4"
+		"lint-staged": "^13.0.3",
```

Comment:
lint-staged v13 has issues with the git-hooks that we are using. Please revert all dependency version changes. Keep "swiper" package. 

```
# pull my changes done in your PR
git pull https://github.com/Im-Kunal-13/walking-pal-web.git  company-hero

# go to first commit
git reset --soft HEAD~2

#unstage those 2 files
git reset HEAD -- package-lock.json
git reset HEAD -- package.json

yarn add swiper

git add .

git commit -m "feat(company): hero section"

git push -f https://github.com/Im-Kunal-13/walking-pal-web.git  company-hero 

```

Reply:
> 

Ok. On it